.github/workflows/pr-build.yml

```diff
@@ -35,54 +35,63 @@ jobs:
         with:
           image: hypertrace/hypertrace-ingester
           output-mode: github
+          category: hypertrace-ingester
```

Comment:
Added to fix this issue: https://github.com/hypertrace/hypertrace-ingester/actions/runs/5452280343/jobs/9919573254?pr=396

Reply:
Raised separate a PR for it take an effect under `pull_request_traget` : https://github.com/hypertrace/hypertrace-ingester/pull/397